### DISCUSSION TITLE
KERI Implementor's Call 08-27-2024

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#96] [Meetings]
Zoom

https://us06web.zoom.us/j/81679782107?pwd=cTFxbEtKQVVXSzNGTjNiUG9xVWdSdz09

Reports

KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

Discussion:
 
@dhh1128 
> All: I heard through other Provenant folks that there's pressure building for a more definitive upgrade to version 2.0 (of CESR specifically, I believe). I think some of it may be connected to issues/842 and Phil's fix in pull/844? Maybe other things as well; I heard Sam mention this as an important objective of a new KERI initiative, too.

 >I think I know some of the reasons for wanting this, but I don't think I know all of them. I'm also worried that this could have ramifications for existing ACDCs and wallets and could be a challenge that takes a lot of management. Moving from cmdline wallets to the keria/signify architecture took Provenant almost a year, and we made some unfair pleas for help of Phil and others along the way. It was so painful and disruptive to our business that it makes me wonder if Provenant should just plan to backport all fixes to a 1.x branch for a while -- though I'm sensitive to how bad it would be to divide our energies. I'm sure Kevin Griffin and crew at GLEIF are counting the cost of a 2.0 upgrade, and I'd be curious to hear their analysis. And of course, the analysis of others as well.

> This is not advocacy against a migration, but rather a bookmark in my desire to discuss further. I have missed 2 or 3 community meetings in a row, but other Provenant people have been there. I'll try to make the next one; maybe we can discuss further there? Or do we need a dedicated meeting?

**Comment by 2byrds** [comment]
Polaris-Web:
@HunainSajid will provide an update on polaris-web as it approaches a new release

**Comment by kentbull** [comment]
### Signify-TS

- https://github.com/WebOfTrust/signify-ts/pull/272
  - Good movement forward. Thanks Fergal!
- https://github.com/WebOfTrust/signify-ts/pull/252
- https://github.com/WebOfTrust/signify-ts/pull/273

### CESRide

No new items in the last two weeks.

**Reply by iFergal** [reply]
Signify-TS new PR: https://github.com/WebOfTrust/signify-ts/pull/273 - small one

**Comment by iFergal** [comment]
**Question regarding KERIA:** The long running operations have a name/ID of `f"{typ}.{oid}"`.

The `oid` for exn messages, at least IPEX is the identifier prefix - this is problematic. In a multi-sig, this exn might take a while to complete (others to sign), so I can only act on one IPEX item at a time or I overwrite the latest operation?

**Reply by 2byrds** [reply]
@iFergal will create a test to reproduce this.
Per our meeting discussion, perhaps use the SAID of the exn.

**Comment by rodolfomiranda** [comment]
KERIA:
Open PRs:
- https://github.com/WebOfTrust/keria/pull/282
- https://github.com/WebOfTrust/keria/pull/281
- https://github.com/WebOfTrust/keria/pull/280
- https://github.com/WebOfTrust/keria/pull/275
- https://github.com/WebOfTrust/keria/pull/272
- https://github.com/WebOfTrust/keria/pull/267

New Issues:
- https://github.com/WebOfTrust/keria/issues/283

**Comment by rodolfomiranda** [comment]
I've a question regarding this PR: 
https://github.com/WebOfTrust/keripy/pull/810

**Comment by pfeairheller** [comment]
### Signfypy

* No issues or PRs
